todoapp/TODOapp/app/src/main/java/com/nearsoft/androidschool/todoapp/activities/detail/DetailActivity.java

```diff
@@ -248,4 +265,23 @@ public void onRequestPermissionsResult(int requestCode, @NonNull String[] permis
             locationSwitch.setChecked(false);
         }
     }
+
+    public void prepareNotification(Notification notification, Date date){
+        Intent notificationIntent = new Intent(this, NotificationPublisher.class);
+        notificationIntent.putExtra("notification id", 1);
```

Comment:
do not use hardcoded string, use a string constant here NOTIFICATION_EXTRA = "notification"